gm4_combat_expanded/data/gm4_combat_expanded/loot_tables/armor/identification/modifier/totem.json

```diff
@@ -188,6 +174,7 @@
         {
           "function": "minecraft:set_name",
           "entity": "this",
+          "target": "item_name",
```

Comment:
No other module uses `item_name` as of now. Are you sure that these items will still stack with existing items?

Imo we should keep using `custom_name` for the initial update, and then update all modules to switch to `item_name` when appropriate. This would also involve removing the unnecessary `"italic": false`, and using the `rarity` component instead of text component colors.

Reply:
These are all armor pieces, so there is no stacking to begin with. `item_name` seemed like the preferred thing to change